.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Would you mind updating the README to remove this section or mention that it is included by default?

Reply:
👍 Sure! Updated that.